eve/main.yml

```diff
@@ -1466,3 +1469,277 @@ stages:
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Activate example-solution
+          env:
+            SOLUTION_VERSION: "%(prop:example_solution_version)s"
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo bash
+            /var/tmp/metalk8s/solutions.sh activate --name example-solution
+            --version "${SOLUTION_VERSION}" --verbose
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Create example-solution Environment
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo bash
+            /var/tmp/metalk8s/solutions.sh create-env --name example-environment
+            --verbose
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      # Taint bootstrap node to allow deployment of example-solution-operator
+      - ShellCommand:
+          name: Taint Bootstrap node before deploying example-solution-operator
+          env:
+            KUBECONF: "--kubeconfig /etc/kubernetes/admin.conf"
+          command: |
+            ssh -F ssh_config bootstrap --
+            bash -c '
+            sudo kubectl ${KUBECONF} taint nodes --all node-role.kubernetes.io/infra:NoSchedule-
+            sudo kubectl ${KUBECONF} taint nodes --all node-role.kubernetes.io/bootstrap:NoSchedule-
+            '
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Deploy example-solution
+          env:
+            SOLUTION_VERSION: "%(prop:example_solution_version)s"
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo bash
+            /var/tmp/metalk8s/solutions.sh add-solution --name
+            example-environment --solution example-solution --version
+            "${SOLUTION_VERSION}" --verbose
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
+          haltOnFailure: true
+      - ShellCommand:
+          name: Copy example-solution CRs to Bootstrap node
+          env:
+            SSH_CONFIG: >-
+              eve/workers/openstack-multiple-nodes/terraform/ssh_config
+            HOST_CR_PATH: "examples/metalk8s-solution-example/example-operator/deploy/crds/examplesolution_v1alpha1_versionserver_cr.yaml"
+            BOOTSTRAP_CR_PATH: "/tmp/examplesolution_v1alpha1_versionserver_cr.yaml"
+          command: >
+            scp -F $SSH_CONFIG "${HOST_CR_PATH}" bootstrap:"${BOOTSTRAP_CR_PATH}"
+          haltOnFailure: true
+      - ShellCommand:
+          name: Apply example-solution CRs on Bootstrap node
+          command: >
+            ssh -F ssh_config bootstrap
+            sudo kubectl --kubeconfig /etc/kubernetes/admin.conf apply -f
+            /tmp/examplesolution_v1alpha1_versionserver_cr.yaml
```

Comment:
This CR must be put in the right namespace `example-environment`.

Reply:
https://github.com/scality/metalk8s/pull/2321/files#diff-fa1f5352a12d0b35290d51280da84271R5

Reply:
Oh! Yes! :p